api/server.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const cors = require('cors');
+const helmet = require('helmet');
+require('dotenv').config();
+
+
+const server = express();
+
+server.use(helmet());
+server.use(cors());
+server.use(express.json());
+
+server.use('/', (req, res)=>{
+  res.status(200).json({message: `${process.env.SITE_NAME} ${process.env.NODE_ENV} backend API`})
+})
```

Comment:
the above will show if you are in test, production, or development environments based on NODE_ENV, and should display the SITE_NAME as well. this is a good way to know if your environment variables or .env file are/is working